src/LineLogin.js

```diff
@@ -7,36 +7,41 @@ class LineLogin {
    * Logs the user.
    */
   login = () => {
-    return LineLoginManager.login();
+    const data = {
+      "data" = "data"
+    }
+
+    if (data.data == 1) return true
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/manual/elmotan95/react-native-line/pullRequest?prid=4781833)